packages/manager/modules/pci/src/projects/project/kubernetes/components/autoscaling/autoscaling.html

```diff
@@ -0,0 +1,100 @@
+<!--Autoscaling stepper head-->
+<div>
+    <!--Description and Link-->
+    <div class="mb-4">
+        <span
+            data-translate="kubernetes_add_node_pool_autoscaling_description"
+        ></span>
+        <a
+            data-ng-href="{{::$ctrl.autoscalingInfoLink}}"
+            data-ng-click="$ctrl.onLinkClicked({link: $ctrl.autoscalingInfoLink})"
+            target="_blank"
+            rel="noopener"
+        >
+            <span
+                data-translate="kubernetes_add_node_pool_autoscaling_description_link"
+            ></span>
+            <span
+                class="oui-icon oui-icon-external_link"
+                aria-hidden="true"
+            ></span>
+        </a>
+    </div>
+
+    <!--DesiredNodes-->
+    <div class="mb-4" data-ng-if="$ctrl.displayDesiredNodes">
+        <oui-field
+            data-label="{{ ::'kubernetes_add_node_pool_autoscaling_desired_nodes' | translate }}"
+            data-size="xl"
+        >
+            <oui-numeric
+                id="node-pool-count"
+                name="node-pool-count"
+                min="$ctrl.model.nodes.lowest.value"
+                model="$ctrl.model.nodes.desired.value"
+                max="$ctrl.model.nodes.highest.value"
+                required
+            >
+            </oui-numeric>
+        </oui-field>
+    </div>
+
+    <!--Autoscaling-->
+    <div class="mb-4">
+        <oui-field
+            data-label="{{:: 'kube_common_node_pool_autoscaling_autoscale' | translate}}"
+            data-size="xl"
+        >
+            <oui-switch
+                model="$ctrl.model.autoscale"
+                on-change="$ctrl.onAutoscaleChanged({status: modelValue})"
+            >
+            </oui-switch>
+            <strong
+                data-translate="{{'kube_common_node_pool_autoscaling_toggle_' + $ctrl.model.autoscale | translate }}"
+            ></strong>
+        </oui-field>
+    </div>
+</div>
+
+<!--Autoscaling steeper body-->
+<div data-ng-if="$ctrl.displayNodePoolSizing">
+    <!--Autoscaling lowest config-->
+    <div class="mb-4">
+        <oui-field
+            data-label="{{ ::'kube_common_node_pool_autoscaling_size_lowest' | translate }}"
+            data-size="xl"
+        >
+            <oui-numeric
+                id="autoscaling-node-pool-lowest-count"
+                disabled="!$ctrl.model.autoscale"
+                name="autoscaling-node-pool-lowest-count"
+                min="$ctrl.model.nodes.lowest.min"
+                model="$ctrl.model.nodes.lowest.value"
+                max="$ctrl.model.nodes.desired.value"
+                on-change="!$ctrl.isEditMode ? $ctrl.nodeCount = modelValue : null"
```

Comment:
Maybe logic could be extracted to a function

Reply:
Changed.